public/index.html

```diff
@@ -1,43 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
💬 
한국어 서비스이기 때문에 약간의 디테일을 살려 이렇게 바꾸면 어떨까요?? 👀 
```suggestion
<html lang="ko">
```